eos/form-factors/analytic-b-to-gamma-qcdf.cc

```diff
@@ -241,15 +241,23 @@ namespace eos
         // cf. [BBJW:2018A], Eq. (4.10)
         const double term_soft_nlo = 0.0;
 
-        // cf. [BBJW:2018A], Eq. (4.14)
-        const double omega_cut = s_0 / (2.0 * Egamma);
-        const double sigma = 2.0 * Egamma / M2;
-
-        const double term_soft_tw_3_4 = e_u * m_B * f_B / (4.0 * Egamma * Egamma) * (
-                2.0 * Egamma / (m_rho * m_rho) * std::exp(m_rho * m_rho / M2)
-                    * lapltr_incomplete_dsigma(omega_cut, sigma) / (-sigma)
-                - norm_incomplete(omega_cut)
-            );
+        // For consistency with the current implementation, the twist-3,4 contribution
```

Comment:
This could have been achieved by introducing a further ``switch_soft_tw34`` or similar, could it not?

Reply:
This is a good option, switching it off per default. Do you think a comment here in the code, that the contribution is incomplete, is sufficient?